web/app.fluidity.money/app/routes/$network/dashboard/airdrop/common.tsx

```diff
@@ -198,8 +231,325 @@ const StakingStatsModal = ({
   );
 };
 
-const StakeNowModal = () => {
-  return <></>;
+interface IStakingNowModal {
+  fluidTokens: Array<AugmentedToken>;
+  baseTokens: Array<AugmentedToken>;
+}
+
+type StakingAugmentedToken = AugmentedToken & {
+  amount: string;
+};
+
+const StakeNowModal = ({ fluidTokens, baseTokens }: IStakingNowModal) => {
+  const [fluidToken, setFluidToken] = useState<StakingAugmentedToken>({
+    ...fluidTokens[0],
+    amount: "",
+  });
+  const [baseToken, setBaseToken] = useState<StakingAugmentedToken>({
+    ...baseTokens[0],
+    amount: "",
+  });
+  const [stakingDuration, setStakingDuration] = useState(31);
+
+  // Staking Multiplier Equation
+  // Reference: https://www.notion.so/fluidity/Fluidity-Airdrops-Loot-bottles-5d1bcf986d5a45a7a29bfc200fbfbb20?pvs=4#f47a040e934a41bbbb1793a7349146b6
```

Comment:
maybe reference a public doc if one exists (cc @af-afk)

Reply:
Also, this is an internal doc @glo12345